src/Components/ApiRequest/ApiRequest.js

```diff
@@ -0,0 +1,11 @@
+const ApiResults = location => {
+  const api = Object.freeze({
+    API_KEY: '&appid=0d26436ded0fcb43c6227b80b9ae1a69',
+    API_URL: 'http://api.openweathermap.org/data/2.5/weather?q=',
+  });
+  const data = `${api.API_URL}${location}${api.API_KEY}`;
```

Comment:
tas I guess this is the endpoint from where you get the weather data, so it should be called
this should be also stored in 
common/variables.js file 


const OPEN_WEAHTER_ENDPOINT = `${api.API_URL}${location}${process.env.OPEN_WEAHTER_API_KEY}`;

Reply:
I stored that url on .env with api key